modules/administration-guide/partials/proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc

```diff
@@ -45,22 +44,18 @@ metadata:
   name: devworkspace-controller-metrics-binding
 subjects:
   - kind: ServiceAccount
-    name: __<ServiceAccount_name>__ <1>
-    namespace: __<Prometheus_{orch-namespace}>__ <2>
+    name: __<ServiceAccount_name_associated_with_the_Prometheus_Pod>__
+    namespace: __<Prometheus_namespace>__
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: devworkspace-controller-metrics-reader
 ----
-
-<1> Name of the `ServiceAccount` associated with the Prometheus Pod.
-<2> The {orch-namespace} in which Prometheus is running.
-
 ====
 
-. Configure Prometheus to scrape metrics from the `8443` port exposed by the `devworkspace-controller-metrics` service, and `9443` port exposed by the `devworkspace-webhookserver` service.
+. Configure Prometheus to scrape metrics from port `8443` exposed by the `devworkspace-controller-metrics` Service and from port `9443` exposed by the `devworkspace-webhookserver` Service.
 +
-.Prometheus configuration example
+.Prometheus configuration
```

Comment:
Just to confirm that this is the same also for the CRW.

Reply:
Yes, same for CRW

Reply:
OK, thank you. Feel free to resolve this conversation.